sparse_strips/vello_cpu/src/fine/highp/mod.rs

```diff
@@ -99,7 +100,12 @@ impl<S: Simd> FineKernel<S> for F32Kernel {
         alphas: Option<&[u8]>,
     ) {
         if let Some(alphas) = alphas {
-            alpha_fill::alpha_composite_solid(simd, dest, src, alphas);
+            alpha_fill::alpha_composite_solid(
+                simd,
+                dest,
+                src,
+                alphas.chunks_exact(4).map(|c| [c[0], c[1], c[2], c[3]]),
```

Comment:
Is it possible to use bytemuck to zero cost reinterpret `alphas` (here and in the below instances)?